Sandbox/Program.cs

```diff
@@ -85,6 +85,8 @@ static void Main(string[] args)
             productionQueueBot.Add(TerranBuildingType.OrbitalCommand);
             productionQueueBot.Add(TerranBuildingType.SupplyDepot);
 
+            //var bot = new CompositeBot(new IBot[] { new ZergRushBot(), new SpawnLarvaBot(), militaryBot});
```

Comment:
?

Reply:
I pulled the MilitaryBot out of the ZergRushBot. I ran a couple games with the increasing wave size like you had it and it seemed to be much better.

I also removed Queen behavior from ZergRushBot. I figured those two factors made it a decent example of a composite bot- I don't think it will beat BenchmarkBot without incorporating more advanced (vespene) logic anyway, so its role seems to have evolved (get it?) to serve as the bridge to more intricate concepts.